packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -361,4 +379,28 @@ module.exports = class Checkout extends Page {
         this.goToPaymentButton.scrollIntoView();
         this.goToPaymentButton.click();
     }
+
+
+    /**
+    * @description
+    * Clicks accordion header of delivery notes and sets the value of the text area
+    *
+    * @param {String} noteType The note type to be opened and populated
+    * @param {String} note The value of the note
+    */
+    expandAndPopulateNote (noteType, note) {
+        switch (noteType) {
+            case 'delivery':
+                this.deliveryAccordionHeader.click();
+                this.fields.deliveryNote.input.setValue(note);
+                break;
+            case 'kitchen':
+                this.kitchenAccordionHeader.click();
+                this.fields.kitchenNote.input.setValue(note);
+                break;
+            default:
+                this.restaurantAccordionHeader.click();
+                this.fields.restaurantNote.input.setValue(note || 'restaurant note');
+        }
+    }
```

Comment:
Not a huge fan of this, personally.
How about we pass the accordion and note to be used by parameters? Something like:

```
expandAndPopulateNote (header, noteType, note) {
  this[header].click();
  this.fields[noteType].input.setValue(note);
}

expandAndPopulateNote('deliveryAccordionHeader', 'deliveryNote', 'My note');
```

Reply:
Agreed. Changed now.